pkg/provisioner/ironic/ironic.go

```diff
@@ -1221,6 +1221,59 @@ func (p *ironicProvisioner) Prepare(data provisioner.PrepareData, unprepared boo
 	return
 }
 
+func (p *ironicProvisioner) getConfigDrive(data provisioner.ProvisionData) (result nodes.ConfigDrive, err error) {
```

Comment:
Elsewhere in this file a return parameter named `result` is usually a `provisioner.Result`... it might be less confusing to name this `configDrive`.

Reply:
I can follow-up if this PR is merged the way it is.